packages/back-end/types/datasource.d.ts

```diff
@@ -166,8 +167,10 @@ export type DataSourceEvents = {
 
 export type DataSourcePipelineSettings = {
   allowWriting?: boolean;
+  writeDatabase?: string;
```

Comment:
This is configurable on the front-end, but it doesn't appear to be used on the back-end at all.

Reply:
Hmm, I went to add the ability to change database/project(BQ)/catalog(databricks) but then never followed through I guess. Well, it's working now.

You can override the default database by setting this field and it passes through. If it is unset, it accepts the default you have set in your connection settings.

Reply:
Confirmed it works with old settings for snowflake and BigQuery (blank continues to accept default database) and works for databricks.